server/models/token.ts

```diff
@@ -0,0 +1,33 @@
+import * as Sequelize from 'sequelize';
+import { AddressAttributes } from './address';
+import { OffchainCommunityAttributes } from './offchain_community';
+import { ChainAttributes } from './chain';
+
+export interface TokenAttributes {
+  name: string;
+  address: string;
+  symbol: string;
+  decimals: number;
+}
```

Comment:
Should include createdAt and updatedAt here as well, and below in the model definition.